Sources/Benchmark/BenchmarkCommand.swift

```diff
@@ -15,6 +15,12 @@
 import ArgumentParser
 import Foundation
 
+public enum Column: String, ExpressibleByArgument, Decodable {
```

Comment:
I think we'll want to allow custom metrics (https://github.com/google/swift-benchmark/issues/11) that are defined on a per-benchmark basis. As a result, I think having an enum is not the right design here, as an enum induces a 'closed set' vs a desired 'open set'. Does that make sense?